packages/fast-components-styles-msft/src/utilities/color/accent-foreground.ts

```diff
@@ -19,61 +20,40 @@ import {
     backgroundColor,
 } from "../design-system";
 
-const accentForegroundAlgorithm: (
-    designSystem: DesignSystem,
+function accentForegroundAlgorithm(
     contrastTarget: number
-) => SwatchFamily = (
-    designSystem: DesignSystem,
-    contrastTarget: number
-): SwatchFamily => {
-    const accentPalette: Palette = palette(PaletteType.accent)(designSystem);
-    const indexes: {
-        rest: number;
-        hover: number;
-        active: number;
-    } = findAccessibleAccentSwatchIndexs(
-        designSystem,
-        contrastTarget,
-        backgroundColor(designSystem),
-        {
-            rest: accentForegroundRestDelta(designSystem),
-            hover: accentForegroundHoverDelta(designSystem),
-            active: accentForegroundActiveDelta(designSystem),
-        }
-    );
+): DesignSystemResolver<SwatchFamily> {
+    return (designSystem: DesignSystem): SwatchFamily => {
+        const accentPalette: Palette = palette(PaletteType.accent)(designSystem);
+        const indexes: {
+            rest: number;
+            hover: number;
+            active: number;
+        } = findAccessibleAccentSwatchIndexs(
```

Comment:
isn't this deprecated? I thought you now have `findAccessibleAccentSwatchIndexes`